src/coreclr/jit/flowgraph.cpp

```diff
@@ -3026,7 +3026,7 @@ void Compiler::fgInsertFuncletPrologBlock(BasicBlock* block)
     assert(nullptr == fgGetPredForBlock(block, newHead));
     fgAddRefPred(block, newHead);
 
-    assert((newHead->bbFlags & BBF_INTERNAL) == BBF_INTERNAL);
+    assert(newHead->CheckFlag(BBF_INTERNAL));
```

Comment:
This can probably be `HasFlag`. It's equivalent if the flag being checked is a single bit. That's probably the case for most/all of the uses of `CheckFlag` with a single flag.

Reply:
Thanks for pointing this out. I was able to get rid of `CheckFlag` altogether.